src/pages/_app.tsx

```diff
@@ -79,7 +79,11 @@ function MyApp(props: AppProps) {
         <meta name="twitter:title" content="🥞 PancakeSwap - A next evolution DeFi exchange on BNB Smart Chain (BSC)" />
         <title>PancakeSwap</title>
         {(Component as NextPageWithLayout).mp && (
-          <script src="https://public.bnbstatic.com/static/js/mp-webview-sdk/webview-v1.0.0.min.js" id="mp-webview" />
+          <script
+            src="https://public.bnbstatic.com/static/js/mp-webview-sdk/webview-v1.0.0.min.js"
+            id="mp-webview"
+            defer
```

Comment:
Should we using `async` or `defer `

Reply:
Mp team ask for not sync or defer this script. Let's add a commment to ignore this eslint. It only affects to mp page